src/components/box/_box.scss

```diff
@@ -1,8 +1,10 @@
 $boxBorderColor: $graySecondary;
 $boxBorderRadius: 22px;
-$boxBorderSize: rhythm(1/8);
+$boxBorderSize: rhythm(1 / 8);
```

Comment:
I agree looks weird but it treats it as equation :( maybe 0.125 would be better ??